### ISSUE TITLE
Issue with persistent login

### ISSUE THREAD

**Issue by koulomesh** [Shopify/checkout-sheet-kit-react-native#84]
**Describe the bug**

While checking out from the cart, the process we follow is creating a multipass link in which the input is checkoutUrl from cart storefront graphql request. When we open that url inside the checkout sheet, it works fine. The cart gets associated with the checkout. But on subsequent loads, it asks for web login inside the sheet.

**To Reproduce**

**Expected behavior**

**Screenshots/Videos/Log output**

**Versions (please complete the following information):**

**Shop domain**

**Additional context**

**Comment by markmur** [comment]
Hey @koulomesh, multipass tokens typically have a short TTL. Are you providing a TTL when you generate the access token?

It's likely that you will need to regenerate a new token if the first has expired. When you reload checkout, are you calling present with a checkout URL? or the multipass URL?

**Comment by markmur** [comment]
This means that you have 15 mins to hit the multipass URL and once you do the token will expire. If your customers close the checkout and reopen it, you will likely need to recreate a new multipass URL to login.

If you're using [preloading](https://github.com/Shopify/checkout-sheet-kit-react-native?tab=readme-ov-file#preloading), the checkout view will be cached for 5 minutes, meaning it will maintain the state across opens/closes during that time. If not, you will need to regenerate the multipass URL each time you present.

**Comment by koulomesh** [comment]
I am doing the same, generating a new multipass URL everytime the user presents the checkout sheet. Still, the login issue persists. In one session, it is auto logged in, then again it shows website's login route before proceeding ahead.

**Comment by markmur** [comment]
@koulomesh an incident was resolved relating to multipass logouts. Can you still reproduce?

**Comment by koulomesh** [comment]
Yeah, it is working fine now. But sometimes, just empty screen shows up in the bottomsheet.
